client/app/src/filters/filters.js

```diff
@@ -1,7 +1,7 @@
 ;(function () {
   'use strict'
 
-  angular.module('arkclient.filters')
+  angular.module('arkclient.filters' )
```

Comment:
Does the linter flag that additional space? (Also, on line 1 can you remove that semicolon -- unless it's necessary for some reason)

Reply:
I'm not sure if the semicolon is necessary, but it's in most of the files so I think it's on purpose for some reason.